src/coroutines/context/setup_stack/setup_stack_x64.hpp

```diff
@@ -0,0 +1,26 @@
+#pragma once
+
+#include <yaclib/coroutines/context/stack_view.hpp>
+
+namespace yaclib {
+
+using Trampoline = void (*)(void* arg);
+
+struct YaclibFiberMachineContext {
```

Comment:
You should use 
`void* registers[YACLIB_CONTEXT_SIZE];`
And use `switch_context_x86_64.hpp` for addressing